shell/components/form/Error.vue

```diff
@@ -0,0 +1,37 @@
+<script>
+export default {
+  props: {
+    value: {
```

Comment:
Please add jsdoc annotation for new components.

Reply:
This is a pretty straightforward component and I'm not certain we've established a standard for commenting every prop on new components just yet. I'll keep this in mind for more complex components in the future though.